src/lib/config/index.ts

```diff
@@ -252,6 +252,19 @@ export class ShellConfig {
 
     return this.endpoint!.makeScopedEndpoint(database, opts.role);
   };
+
+  /**
+   * Saves the project config, if present.
+   */
+  saveProjectConfig() {
+    this.projectConfig?.save(this.projectConfigFile()!);
+  }
```

Comment:
wondering if we should error if this is ever attempted without a project config?

Reply:
i don't really mind either way. the `?.` made it shorter to just do nothing if you don't have a project config, but I don't really have a preference.